src/modules/Auth/hooks/useLogoutUser.ts

```diff
@@ -0,0 +1,25 @@
+import { useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useLogoutUserMutation } from '@/queries/auth/authMutations';
+import { AUTH_PAGE } from '../../../constants';
+
+type UseCreateUserParams = {
+  onError?: () => void;
+};
+
+export const useLogoutUser = ({ onError }: UseCreateUserParams) => {
```

Comment:
You didn't pass this params in place where you invoke this hook, so delete this params